app/assets/javascripts/embargo_reason_field.js

```diff
@@ -0,0 +1,11 @@
+$( document ).ready(function() {
+  $('#embargo_reason_select').change(function() {
```

Comment:
If the user leaves the default selection without triggering a "change" on the select, what would happen?